oak_functions/loader/src/grpc.rs

```diff
@@ -81,13 +69,16 @@ pub async fn create_and_start_grpc_server<F: Future<Output = ()>>(
         ),
     );
 
+    let request_handler =
+        async move |request: Vec<u8>| handle_request(wasm_handler, policy, request.to_vec()).await;
+
     // A `Service` is needed for every connection. Here we create a service using the
     // `wasm_handler`.
     tonic::transport::Server::builder()
-        .add_service(GrpcHandlerServer::new(FunctionsServer {
-            wasm_handler,
-            policy,
-        }))
+        .add_service(RemoteAttestationServer::new(
+            AttestationServer::create(tee_certificate, request_handler)
+                .context("Couldn't create proxy")?,
```

Comment:
Should it be called a proxy?
Maybe `Couldn't create remote attestation server` ?

Reply:
Done.